benchmarks/statevec.py

```diff
@@ -99,7 +99,9 @@ def simple_random_circuit(nqubit, depth):
 # to transpile into a measurement pattern.
 
 
-def translate_graphix_rc_into_paddle_quantum_circuit(graphix_circuit: Circuit) -> PaddleCircuit:
+def translate_graphix_rc_into_paddle_quantum_circuit(
+    graphix_circuit: Circuit,
+) -> PaddleCircuit:
```

Comment:
This kind of changes are probably due to `black` not having been called with `-l 120`. They should not appear in the PR.